turbinia/turbiniactl.py

```diff
@@ -358,10 +358,16 @@ def main():
   elif args.command == 'googleclouddiskembedded':
     is_cloud_disk = True
     args.name = args.name if args.name else args.disk_name
+    parent_evidence_ = evidence.GoogleCloudDisk(
+        name=args.name, disk_name=args.disk_name, project=args.project,
+        mount_partition=args.mount_partition, zone=args.zone,
+        source=args.source)
+    # TODO: have a way to provide the partition in the parent evidence to get
```

Comment:
Can we for now just add a `--embedded_mount_partition` flag (or something similar), and use that to set the `GoogleCloudDiskRawEmbedded.mount_partition`?  And then we can keep `--mount_partition` flag to set the inner `GoogleCloudDisk.mount_partition`.

Reply:
Done.

The new attribute will be used in the next PR (the one that will mount filesystems)